docs/can-guides/contribute/site-style-guide.md

```diff
@@ -0,0 +1,87 @@
+@page guides/contributing/site-style-guide Site Style Guide
+@parent guides/contribute 7
+
+@description
+This is the CanJS Site Style Guide
```

Comment:
This is mostly just repeating the title. Most of the other pages have something like “Learn about X.” I like to think of these descriptions in the context of when someone searches… if someone searches for “style” or “guide” and this is one of the results, will they understand what the page is? I also like to put in other “keywords” that someone might search for that should bring up this page.